src/sentry/tasks/collect_project_platforms.py

```diff
@@ -39,5 +39,5 @@ def collect_project_platforms(**kwargs):
 
     # remove (likely) unused platform associations
     ProjectPlatform.objects.filter(
-        last_seen__lte=now - timedelta(days=30),
+        last_seen__lte=now - timedelta(days=90),
```

Comment:
As a follow up, we should add an index to this column. I didn't realize we were doing queries like this.